tests/Unit/API/Site/Controllers/MerchantCenter/AttributeMappingControllerTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\Tests\Unit\API\Site\Controllers\MerchantCenter;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\MerchantCenter\AttributeMappingController;
+use Automattic\WooCommerce\GoogleListingsAndAds\Product\AttributeMappingHelper;
+use Automattic\WooCommerce\GoogleListingsAndAds\Tests\Framework\RESTControllerUnitTest;
+
+/**
+ * Test suit for AttributeMappingController
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\Tests\Unit\API\Site\Controllers\MerchantCenter
+ * @group AttributeMapping
```

Comment:
I Agree that we can a filter with the naming but for sets of 3 or more files it starts to be big, plus the potential typo 

With group it's just `vendor/bin/phpunit --group="AttributeMapping"` which is very convenient at least for me.

Since it's not harmful,  I guess we can keep it?

Reply:
Yes, by all means keep it (especially if it's convenient for you). I was merely pointing out how I run tests since the group isn't always added.